src/scores/categorical/contingency_impl.py

```diff
@@ -449,7 +449,7 @@ def probability_of_false_detection(self) -> xr.DataArray:
 
     def success_ratio(self) -> xr.DataArray:
         """
-        Identical to :py:func:`precision`.
+        Identical to :py:func:`precision` and :py:func:`positive_predictive_value`
```

Comment:
```suggestion
        Identical to :py:func:`precision` and :py:func:`positive_predictive_value`.
```
It looks like this full stop wasn't added? (But if I am wrong, then no worries!)